functions/index.js

```diff
@@ -52,6 +53,13 @@ exports.dpeCronSTTJobChecker = functions
     sttChecker.createHandler(admin, config.aws.api.transcriber, context)
   );
 
+exports.dpeCronExpiredMediaChecker = functions
+  .runWith(maxRuntimeOpts)
+  .pubsub.schedule("every 24 hours")
+  .onRun(() => {
```

Comment:
I think this should be `.onRun((context) => dpeCronExpiredMediaChecker.createHandler(bucket, admin);
  );` (note I've removed the curly brackets), looking at how `exports.dpeCronSTTJobChecker` is written above - see also https://firebase.google.com/docs/reference/functions/providers_pubsub_.schedulebuilder#onrun.

.onRun expects a parameter of a function which returns a Promise - so change your dpeCronExpiredMediaChecker.createHandler function to return a promise and that error should go away.